edx_proctoring/views.py

```diff
@@ -371,23 +371,26 @@ def get(self, request):
 
         attempts = ProctoredExamStudentAttempt.objects.get_proctored_practice_attempts_by_course_id(course_id, [user])
 
-        if len(attempts) == 0:
-            # If there are no attempts in the current course, check for a verified attempt in another course
+        # Default to the most recent attempt in the course if there are no verified attempts
+        relevant_attempt = attempts[0] if attempts else None
```

Comment:
#NIT I feel like this set of logic to select the correct attempt to use can be refactored, so the logic at line 487 can share the same logic. Basically, a function that takes in the attempt within the course, and a verified_attempt outside of the current course, and return the attempt to return.

Reply:
Making a note to take a look at this while re-factoring in #792